dev-requirements.txt

```diff
@@ -1,7 +1,7 @@
 requests==2.31.0
-setuptools==69.5.1
-wheel==0.43.0
-twine==5.0.0
+setuptools>=40.8.0
```

Comment:
`setuptools`, `wheel`, and `twine` are essential requirements for this package. Their versions should be fixed in `dev-requirements.txt`. This will assist `dependabot` in updating the file on a new version of any of these libraries.

Reply:
But here is a problem, in python 3.6 we don't support the latest versions of these libraries, what should I do?
as you can see in some failed actions, the limit for fixing the version was the python3.6's not supporting issue.
@sepandhaghighi

Reply:
Applied.